sdk/schemaregistry/Azure.Data.SchemaRegistry/src/Generated/SchemaRestClient.cs

```diff
@@ -174,7 +174,7 @@ internal HttpMessage CreateQueryIdByContentRequest(string groupName, string sche
         /// <param name="schemaContent"> String representation (UTF-8) of the registered schema. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="groupName"/>, <paramref name="schemaName"/>, or <paramref name="schemaContent"/> is null. </exception>
-        public ResponseWithHeaders<SchemaId, SchemaQueryIdByContentHeaders> QueryIdByContent(string groupName, string schemaName, SerializationType serializationType, string schemaContent, CancellationToken cancellationToken = default)
+        public ResponseWithHeaders<SchemaId, SchemaQueryIdByContentHeaders> QueryIdByContent(string groupName, string schemaName, SchemaFormat serializationType, string schemaContent, CancellationToken cancellationToken = default)
```

Comment:
Should we update to use `format` instead of `serializationType` to reflect the `SchemaFormat` name change and to match the `SchemaProperties` use?

Reply:
This is generated code so changing it would require swagger changes.